TestApp/MMB/Sources/MMBViews/MMBFlow.swift

```diff
@@ -0,0 +1,78 @@
+import SwiftUI
+import Navigation
+import MMB
+import UserStorage
+import LoginViews
+
+public struct MMBFlow: View {
+
+    enum Path: Hashable {
```

Comment:
Is there an advantage of having `Path/Destination` separate from `Modal`? If all presentation modifier APIs use the `for:case:` syntax, then all possible destinations of a screen could be in a single `Destination` enum.

The `modal` name might be confusing, as later it could be a place to store popovers, non-modal bottomsheets etc.

Reply:
Suggestion then:

`enum (Navigation)Destination` + `enum PresentedDestination` ? 

- But still, considering all ±6 types of presentations/navitations, a single `Destination` might be more flexible and easier to use.

Reply:
I used the suggested naming - this reads nicely:
```swift
@Published var presented: PresentedDestination?
```

The "problem" with having a single enum for both would be that we would have to define a view for every case in push navigation, even if some of those views are only meant for modal presentation. Switches have to be exhaustive, after all.